helm/provisioner/values.yaml

```diff
@@ -67,6 +69,8 @@ classes:
     fsType: ext4
     # File name pattern to discover. By default, discover all file names.
     namePattern: "*"
+    # Restrict topology of provisioned volumes to specific labels
+    allowedTopologies:
```

Comment:
does this empty setting work? I suppose it should write like this `allowedTopologies: {}`